modules/template/src/main/twirl/scaladex/view/main.scala.html

```diff
@@ -107,32 +107,54 @@
     </header>
     @content
     <footer id="bottom">
-        <div class="container">
-            <div class="row">
-                <div class="col-md-2">
-                    <ul>
-                        <li>
-                            <h5>Scaladex</h5></li>
-                        <li><a href="https://github.com/scalacenter/scaladex/wiki/FAQ">FAQs</a></li>
-                        <li><a href="https://github.com/scalacenter/scaladex/issues/new">Report an Issue</a></li>
-                    </ul>
-                </div>
-                <div class="col-md-2">
-                    <ul class="social">
-                        <li>
-                            <h5>Follow us:</h5></li>
-                        <li><a href="https://twitter.com/scala_index"><i class="fa-brands fa-twitter fa-lg"></i></a></li>
-                        <li><a href="https://github.com/scalacenter/scaladex"><i class="fa-brands fa-github fa-lg"></i></a></li>
-                    </ul>
-                </div>
-                <div class="col-md-8">
+      <div class="container">
+          <div class="footer-top row">
+              <div class="col-md-3">
+                  <ul>
+                      <li><h5>Community</h5></li>
+                      <li><a href="https://discord.com/invite/scala">Discord</a></li>
+                      <li><a href="https://github.com/scalacenter/scaladex/discussions">Discussions</a></li>
+                      <li><a href="https://gitter.im/scalacenter/scaladex">Gitter</a></li>
+                  </ul>
+              </div>
+              <div class="col-md-3">
+                  <ul>
+                      <li><h5>Contribute</h5></li>
+                      <li><a href="https://github.com/scalacenter/scaladex/blob/main/CONTRIBUTING.md">Contribution Guide</a></li>
+                      <li><a href="https://github.com/scalacenter/scaladex/issues/new">Report an Issue</a></li>
+                  </ul>
+              </div>
+              <div class="col-md-3">
+                  <ul>
+                      <li><h5>Scaladex</h5></li>
+                      <li><a href="https://github.com/scalacenter/scaladex/wiki/FAQ">FAQs</a></li>
+                      <li><a href="https://scala-lang.org/conduct/">Code of Conduct</a></li>
+                      <li><a href="https://github.com/scalacenter/scaladex?tab=BSD-3-Clause-1-ov-file">License</a></li>
+                  </ul>
+              </div>
+              <div class="col-md-3">
+                  <ul class="social">
+                      <li><h5>Social</h5></li>
+                      <li><a href="https://twitter.com/scala_index"><i class="fa-brands fa-twitter fa-lg"></i>Twitter</a></li>
+                      <li><a href="https://github.com/scalacenter/scaladex"><i class="fa-brands fa-github fa-lg"></i>Github</a></li>
+                  </ul>
+              </div>
```

Comment:
`Community` and `Social` is a bit redundant. The Github link can be moved to the `Community` section, as suggested above. Also the `Twitter` account is dead so we can remove the link completely.

```suggestion
```